interface/modules/zend_modules/module/PrescriptionTemplates/src/PrescriptionTemplates/Controller/PdfTemplatesController.php

```diff
@@ -35,11 +36,12 @@
  */
 class PdfTemplatesController extends PrescriptionTemplatesController
 {
-    private $renderer;
+    public $renderer;
 
-    public function __construct(PhpRenderer $renderer)
+    public function __construct(ContainerInterface $container)
```

Comment:
I have the same question here... there was no real change made here for why you need the full container instead of the $renderer dependency?  Is this because of some non-public code that you need the full container here?  

I consider injecting the container a bit of an anti-pattern as it hides your dependencies and makes it difficult to test your objects.  But I know its a big debate between the 'service locator' pattern and 'dependency injection' pattern.  However, I'm not seeing this class do much so I'm not sure its that big of a deal.  I'm just trying to understand why you want to make this change?